libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/SetProperty.cs

```diff
@@ -65,6 +65,7 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
 
             dc.GetState().SetValue(this.Property, value);
 
+            dc.GetState().SetValue(DialogPath.Retries, 0);
```

Comment:
> dc.GetState().SetValue(DialogPath.Retries, 0); [](start = 11, length = 47)

??? What does this do, on SetProperty???

Reply:
I've added a work item to review this and assigned to me.

---
In reply to: [360173669](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360173669) [](ancestors = 360173669)